tests/system/test_diags.py

```diff
@@ -239,8 +272,12 @@ def test_enso_diags_scatter_test_ref_yrs(self):
         case_id = 'TREFHT-response-scatter-test-ref-yrs'
         self.check_enso_scatter_plots(case_id)
 
+    # Image diff for `test_lat_lon` fails sporadically, so setting `image_diff=False`.
```

Comment:
For some reason, `test_lat_lon` fails sporadically on the `image_diff` check. I'm not sure why. `diff.show()` doesn't show any differences between `actual_png` and `expected_png`. Sometimes, the check does in fact pass. So, for now, I'm just disabling that check and we can visually inspect that one plot.

Reply:
I suppose actual_png and expected_png are identical in this test? It's kind of odd the check would fail...

Reply:
They appear identical to me, yes. And `diff.show()` just shows a black image, which would indicate nothing is different between them, unless there is an invisible diff, perhaps one pixel? I don't know what's happening exactly.

Reply:
I would suggest to figure out this problem before moving forward. After all, we have largest number of images in lat_lon. And would expect the image checker to return credible results...

Reply:
Alright, I'll do some more investigation. `test_lat_lon_regional` does pass consistently at least.